src/Image.php

```diff
@@ -939,7 +894,7 @@ public static function callBase(
         self::errorIsArray($result);
         $result = self::wrapResult($result);
 
-        self::debug($name, $instance, ['result' => $result]);
+        Main::debug($name, ['result' => $result]);
```

Comment:
Should we log the result instance also? `Main::debug($name, ['instance' => $instance, 'result' => $result]);`